intro.tex

```diff
@@ -38,43 +36,46 @@ \section{Introduction}
 first is the very real risk such studies pose to the privacy of the
 device owner, and the other is the difficulties researchers have
 securing access to devices, and using any data gathered in a
-responsible and ethical manner.
+responsible and ethical manner. These issues can be summarized as follows:
 					
-To begin with, privacy and security threats to mobile devices have
+For device owners, privacy and security threats to mobile devices have
 increased dramatically over the years as potential attackers also seek
 to take advantage of the rich functionality and user experience that
 sensors\footnote{\scriptsize In this work, we broadly define sensors
 as the hardware components that can record phenomena about the
 physical world, such as the WiFi/cellular network, GPS location,
 movement acceleration, etc.} on smartphones and tablets can provide.
-The data acquired through a smartphone's GPS locations,WiFi
+
+\begin{enumerate}
+\item Data acquired through a smartphone's GPS locations,WiFi
 connections, or Bluetooth pairing history can be highly personal,
-exposing sensitive information, such as where a person lives or shops,
-or what sites they visit online e.g., apersonal
-\cite{han2012accomplice}. Some seemingly benign apps \lois{Yanyan-apps SUCH
-AS what?} can provide companies or government agencies access to
-ordinary people's daily activities~\cite{AngryBirds}. Furthermore, a
-malicious party could potentially bypass a device's security
-protections and gain access to user privileges. As a result of such
-threats, many device users, including those who might otherwise enjoy
-participating in studies, are reluctant to volunteer.
-
-Attempting to recruit subjects from a limited pool of willing
-volunteers would be enough to steer most researchers away from using
-smart devices. However, their efforts are further complicated by the
-need to abide by a strict set of guidelines for collecting data to
-preserve the privacy of their human subjects.. For decades,
-universities, government agencies, and private organizations have
-employed groups known as Institutional Review Boards (IRBs) that
+exposing sensitive information, such as where a person lives or shops\cite{han2012accomplice}.
+
+\item  Even seemingly benign applications, such as popular online games downloaded to mobile devices, can
+ "leak" data, such as the model number and screen size of the phone, or worse the age, gender, or location of the user. 
+
+\item Running experiment code poses a real risk to the operation of the device itself, through potential exposure to bugs or other 
+vulnerabilities. It can also seriously interfere with battery life, if the device is accessed too often.
+\end{enumerate}
+
+For potential testbed user, the challenges are equally formidable:
+\begin{itemize}
```

Comment:
Sorry Yanyan. I thought I had all the code in place correctly.
On Dec 1, 2015 4:46 PM, "Yanyan Zhuang" <notifications@github.com> wrote:

> In intro.tex
> <https://github.com/yyzhuang/mobisys16-sensibility/pull/5#discussion_r46343905>
> :
>
> > -smart devices. However, their efforts are further complicated by the
> > -need to abide by a strict set of guidelines for collecting data to
> > -preserve the privacy of their human subjects.. For decades,
> > -universities, government agencies, and private organizations have
> > -employed groups known as Institutional Review Boards (IRBs) that
> > +exposing sensitive information, such as where a person lives or shops\cite{han2012accomplice}.
> > +
> > +\item  Even seemingly benign applications, such as popular online games downloaded to mobile devices, can
> > + "leak" data, such as the model number and screen size of the phone, or worse the age, gender, or location of the user.
> > +
> > +\item Running experiment code poses a real risk to the operation of the device itself, through potential exposure to bugs or other
> > +vulnerabilities. It can also seriously interfere with battery life, if the device is accessed too often.
> > +\end{enumerate}
> > +
> > +For potential testbed user, the challenges are equally formidable:
> > +\begin{itemize}
>
> this should be \begin{enumerate}
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yyzhuang/mobisys16-sensibility/pull/5/files#r46343905>
> .
>